.github/workflows/operator-ci.yml

```diff
@@ -41,11 +41,21 @@ jobs:
           kubernetes version: v1.22.3
           github token: ${{ secrets.GITHUB_TOKEN }}
           driver: docker
+
+      - name: Build the Keycloak Docker image
+        run: |
+          cd quarkus
+          cp dist/target/keycloak-*.tar.gz container/
+          cd container
+          eval $(minikube -p minikube docker-env)
+          docker build --build-arg KEYCLOAK_DIST=$(ls keycloak-*.tar.gz) . -t keycloak:${GITHUB_SHA}
+
       - name: Build , deploy and test operator in minikube
         run: |
           cd operator
           eval $(minikube -p minikube docker-env) 
           mvn clean verify \
               -Dquarkus.container-image.build=true -Dquarkus.container-image.tag=test \
               -Dquarkus.kubernetes.deployment-target=kubernetes \
-              --no-transfer-progress -Dtest.operator.deployment=remote
\ No newline at end of file
+              -Dquarkus.jib.jvm-arguments="-Djava.util.logging.manager=org.jboss.logmanager.LogManager","-Doperator.keycloak.image=keycloak:${GITHUB_SHA}","-Doperator.keycloak.image-pull-policy=Never" \
```

Comment:
Is this really necessary?
```
java.util.logging.manager=org.jboss.logmanager.LogManager
```
If we want to change the logging manager, it should be probably addressed in a separate issue and should be done for the whole operator testsuite.

Reply:
we are not changing anything, on the contrary we are preserving the default:
https://quarkus.io/guides/container-image#quarkus-container-image-jib_quarkus.jib.jvm-arguments

Reply:
But why to explicitly put it into the CI? What is the motivation here?

Reply:
By specifying `-Dquarkus.jib.jvm-arguments` we are overriding a `List` which by default contains only `-Djava.util.logging.manager=org.jboss.logmanager.LogManager`, since I'm not aware of the existence of an operator `+=` we need to redefine the entire `List` to add elements.

Here we are preserving the original value (so that the behaviour of the image remains the same) and add additional values.

Not sure what's the issue here ... Do you see an inconsistency or a problem with preserving the original and default `logging.manager`?

Reply:
@vmuzikar have we cleared up your doubt on this detail?
Do you want to propose a different approach?